### PR TITLE
fix(social-insurance-administration): add scopes

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

* Multiple scopes added, 1 per api.

## Why

* More robust authorization

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Added social insurance API scopes to clients API for enhanced permissions management.

- **Updates**
  - Updated ownership of the `social-insurance-administration` client to include `@island-is/hugsmidjan`.

- **Bug Fixes**
  - Removed the `paymentYears` field from various components and queries to streamline data handling and improve performance.

- **Refactor**
  - Simplified payment plan retrieval and display logic across multiple components for better maintainability and user experience.

### CODE REVIEW

**Review by AndesKrrrrrrrrrrr — APPROVED** [review]
CODEOWNERS LGTM

File: libs/service-portal/social-insurance-maintenance/src/components/PaymentGroupTable/PaymentGroupTable.tsx [hunk 1]
```diff
@@ -53,73 +35,65 @@
...
-          <EmptyTable message={formatMessage(m.noPaymentsFound)} />
+        {!loading && !error && data?.socialInsurancePaymentPlan && (
+          <Table.Body>
+            {data.socialInsurancePaymentPlan?.paymentGroups
+              ?.filter(
+                (pg) => pg.type === SocialInsurancePaymentGroupType.PAYMENTS,
+              )
+              .map((pg, idx) => (
+                <PaymentGroupTableRow
+                  key={`payment-group-idx-${idx}`}
```
[the remark this answers is not in the document]
Comment (thorkellmani) [reply libs/service-portal/social-insurance-maintenance/src/components/PaymentGroupTable/PaymentGroupTable.tsx:46]: There currently isn't a usable id

**Review by jonarnarbriem — APPROVED** [review]

**Review by thordurhhh — APPROVED** [review]
Service portal 🚀

**Review by karenbjorg — APPROVED** [review]
lgtm 😌

**Pull request merged**